src/main/java/uk/gov/hmcts/ccd/domain/model/refdata/CourtVenue.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.ccd.domain.model.refdata;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+
+@Data
+@Value
+@Builder
+@Jacksonized
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class CourtVenue implements Serializable {
+    String courtVenueId;
+    String siteName;
+    String courtName;
```

Comment:
private access modifier? it's debatable :)

Reply:
`@Value` makes the fields private

Reply:
aha.. that's good to know 👍